skimage/transform/_warps.py

```diff
@@ -18,14 +17,27 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
+    if multichannel is not None:
+        return multichannel
+    else:
+        warn('The default multichannel argument (None) is deprecated. '
+             'Specifiy True or False explicitly.')
```

Comment:
I suspect we want multichannel=False in the future but the current behavior is True for 3D images and False for 2D or 4D+ images? I think somehow this has to come across in the deprecation message. Otherwise multichannel would be a required argument after the deprecation, which doesn't sound good. =)

Oh and btw, specify in which version the deprecation will take place.